### ISSUE TITLE
the question about dataset feature

### ISSUE THREAD

**Issue by LithiumZhou** [YuanGongND/whisper-at#29]
Hi Yuan,

The features of the ESC dataset you provided seem to only have whisper-large-v1，But it seems that the provided code includes features from more than one model.
Thanks

**Comment by YuanGongND** [comment]
hi there,

Yes, in the paper we compare the performance of multiple models on ESC-50 (figure 1), the purpose was to show the advantage of Whisper on that. So we may have code of this in this repo.

Since this project is mostly on Whisper, and Whisper has the strongest performance, we only release the feature of Whisper. If you want to get features of other datasets, we do have the code at https://github.com/YuanGongND/whisper-at/tree/main/src/noise_robust_asr/intermediate_feat_extract/esc-50, you can run it by yourself. Hope this helps.

-Yuan

**Comment by LithiumZhou** [comment]
Thank you for your quick answer，
I currently have another question about ESC-50. I performed ESC-50 using your codes but could not reproduce the results in the paper on the two baselines of last-mlp and wa-mlp, with an accuracy of approximately 82%. I would like to know the hyperparameters of this experiment, such as learning rate decay, mixup, and learning rate. 
Your work means a lot to me.

**Comment by YuanGongND** [comment]
Can you reproduce the result of TL-TR methods?

The only hyper parameter we might tuned for different models is the learning rate. You can try 5X 10X times larger or 5X 10X time smaller.

-Yuan

**Comment by YuanGongND** [comment]
And are you using the following code?

https://github.com/YuanGongND/whisper-at/blob/17d94d6acd53866390ce70f95afa13507dcb8aef/src/whisper_at_train/models.py#L112-L132

**Comment by LithiumZhou** [comment]
thank you for your response，

Yes, I can reproduce the results of Tl-TR. I'm also using the same code as you say, and I will continue to try different learning rates to see how it performs on the baseline.
